pom.xml

```diff
@@ -691,26 +687,15 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <version>3.1.1</version>
-                <executions>
-                    <execution>
-                        <phase>validate</phase>
-                        <goals>
-                            <goal>copy</goal>
-                        </goals>
-                        <configuration>
-                            <outputDirectory>${endorsed.dir}</outputDirectory>
-                            <silent>true</silent>
-                            <artifactItems>
-                                <artifactItem>
-                                    <groupId>javax</groupId>
-                                    <artifactId>javaee-endorsed-api</artifactId>
-                                    <version>7.0</version>
-                                    <type>jar</type>
-                                </artifactItem>
-                            </artifactItems>
-                        </configuration>
-                    </execution>
-                </executions>
+            </plugin>
+            <plugin>
+                <groupId>de.qaware.maven</groupId>
+                <artifactId>go-offline-maven-plugin</artifactId>
+                <version>1.2.1</version>
+                <configuration>
+                    <dynamicDependencies>
+                    </dynamicDependencies>
+                </configuration>
```

Comment:
@poikilotherm could this pull request be simplified to only be about adding the new "offline" plugin? What I'm trying to ask is if all the other changes about "endorsed" and "executions" are required for you to get what you immediately want.

Reply:
I had some troubles when I was going to solve this via the normal dependency plugin first.
It crashed due to the missing endorsed API jar, so I tried to eliminate root cause.
If you guys feel better leaving out this commit, no problem. It should be cleaned up anyway, but might be pulled into a separate issue.

Reply:
@poikilotherm yes can you please remove that commit to make the pull request smaller and less controversial? You are welcome to follow up with a future pull request or two for the "endorsed" and "executions" stuff.

Reply:
Done. Will leave it there for now.